packages/app-project/src/components/AuthModal/components/LoginForm/LoginForm.js

```diff
@@ -0,0 +1,122 @@
+import { SpacedText } from '@zooniverse/react-components'
+import counterpart from 'counterpart'
+import { Formik } from 'formik'
+import { Anchor, Box, Button, Heading, Text, FormField, TextInput } from 'grommet'
+import Link from 'next/link'
+import { func } from 'prop-types'
+import React from 'react'
+
+import en from './locales/en'
+
+counterpart.registerTranslations('en', en)
+
+const userNameFieldId = 'LoginForm_login'
+const passwordFieldId =  'LoginForm_password'
+
+const textColors = { dark: 'white', light: 'black' }
```

Comment:
I don't think this is necessary. Looking at the design, it looks like the text color is the standard we've already set in the theme for light, #5C5C5C. @beckyrother could you confirm or make https://projects.invisionapp.com/share/UAN6AP0JTBC#/screens/365296253_00-31_Sign_In inspectable? It's only the share link.

Reply:
I've set this to `light-1` / `dark-5` - without it, all text renders as black in both light and dark modes.

Reply:
also, cc'ing #890